tauri/src/app.rs

```diff
@@ -223,7 +221,7 @@ impl<A: ApplicationExt + 'static> WebviewInitializer<A> for Arc<App<A>> {
       WebviewDispatcher::new(dispatcher, webview_label),
     );
 
-    crate::plugin::created(A::plugin_store(), &manager).await
+    crate::async_runtime::block_on(crate::plugin::created(A::plugin_store(), &manager));
```

Comment:
with this change, if `WebviewManager#create_webview` is called on the invoke handler, the app `might` deadlock since the invoke handler context also uses the async runtime.